notifier.cpp

```diff
@@ -23,8 +23,8 @@ static volatile bool done = false;
 
 
 struct message final {
-    std::string subject;
-    std::string data;
+    std::string subject = "";
+    std::string data = "";
```

Comment:
Default constructor of std::string already do it.

Reply:
nothing to say